cmd/lotus-chainwatch/storage.go

```diff
@@ -294,6 +294,30 @@ create table if not exists miner_info
 		primary key (miner_id)
 );
 
+/*
+* captures chain-specific power state for any given stateroot
+*/
+create table if not exists chain_power
+(
+	stateroot text not null
+		constraint chain_power_pk
+			primary key,
+	baseline_power text not null
+);
+
+/*
+* captures miner-specific power state for any given stateroot
+*/
+create table if not exists miner_power
+(
+	miner_id text not null,
+	stateroot text not null,
```

Comment:
`state_root`

Reply:
We are inconsistent, but I prefer them as two words. I'll change them to match.